CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## Master (Unreleased)
 
-- Fix a false positive for `FactoryBot/FactoryNameStyle` when namespaced models. ([@ydah])
```

Comment:
Why the shuffle?

Reply:
Ah. Sorry. It looks like it was probably rebased and shuffled.